dev-libs/libmelf/libmelf-0.4.0-r1.ebuild

```diff
@@ -21,18 +21,23 @@ PATCHES=(
 	"${FILESDIR}"/${PN}-0.4.0-r1-gcc-makefile-cleanup.patch
 )
 
-src_compile() {
+src_configure() {
+	default
 	append-flags -fPIC
-	emake CC="$(tc-getCC)" OPTFLAGS="${CFLAGS}" || die "emake failed"
+}
+
+src_compile() {
+	emake CC="$(tc-getCC)" OPTFLAGS="${CFLAGS}" -W1,soname,libmelf.so
```

Comment:
Adding the "-W1,soname,libmelf.so" here does not help :-(  Need more instructions

Reply:
Note that the second character in `-Wl,soname,libmelf.so` is a lower-case "L".

There is some documentation for the QA warning: https://dev.gentoo.org/~zmedico/portage/doc/ch07s04.html
For this simpler library, I think, disabling the warning via QA_SONAME should also work.

Reply:
"-W1" => "-Wl", of cource